utils/management_tools/Get_runs_with_most_fields.ipynb

```diff
@@ -0,0 +1,197 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Utility notebook to get the runs with most fields logged in a particular project\n",
+    "\n",
+    "<a target=\"_blank\" href=\"https://colab.research.google.com/github/neptune-ai/examples/blob/main/utils/management_tools/Get_runs_with_most_fields.ipynb\">\n",
+    "  <img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open in Colab\"/>\n",
+    "</a><a target=\"_blank\" href=\"https://github.com/neptune-ai/examples/blob/main/utils/management_tools/Get_runs_with_most_fields.ipynb\">\n",
+    "  <img alt=\"Open in GitHub\" src=\"https://img.shields.io/badge/Open_in_GitHub-blue?logo=github&labelColor=black\">\n",
+    "</a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install -qq -U neptune tqdm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import neptune\n",
+    "import pandas as pd\n",
+    "from tqdm.auto import tqdm\n",
+    "from typing import Optional\n",
+    "import logging"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set Neptune API token"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if \"NEPTUNE_API_TOKEN\" not in os.environ:\n",
+    "    from getpass import getpass\n",
+    "\n",
+    "    os.environ[\"NEPTUNE_API_TOKEN\"] = getpass(\"Enter the Neptune API token you wish to use: \")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Enter project to scan"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "project = input(\"Enter the project to scan in the format workspace/project: \")"
```

Comment:
> Would it be better UX if we asked for the workspace and project name separately?

In the context of this notebook or in general for all our resources?
We would have separate prompts for resources directed to new users. For more experienced ones, a single prompt would mean less interruptions to their workflow.

This notebook is relevant to advanced (heavy) users. So, I am assuming they would be aware of Neptune's conventions.

Reply:
In general, where giving the project path is mandatory. Bear in mind that if users follow our best practices, even advanced users only need to export their project name once. I would not assume that anyone would become that familiar with our conventions, especially if they maybe didn't touch anything config-related for a few months.

So when they must enter the project name (instead of it being read from an environment variable), let's make sure some instructions are included.